src/functions.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+$booksDb = connectToDb('booksDb');
+
+$books = getAllBooks($booksDb);
+
+function displayBooks($books) {
+    $htmlOutput = ''; 
+    foreach ($books as $book) {
+        if (substr($book['authorSecondName'], -1) == "s") {
+            $possessiveMarker = "'";
+        } else {
+            $possessiveMarker = "'s";
+        }    
+        $link = $book['link'];
+        $htmlOutput .= "<section tabindex='0'>";
```

Comment:
I wonder what would happen when you wanted to have the tab index work, and have different numbers? This would leave them all with the same tab index, and might be difficult to change later down the line as stands